xml/operations-objectstorage-validating_swift_recon.xml

```diff
@@ -37,11 +37,11 @@
   <para>
    The following command retrieves and displays disk usage information:
   </para>
-<screen>$ sudo swift-recon --diskusage</screen>
+<screen>&prompt.sudo;swift-recon --diskusage</screen>
   <para>
    For example:
   </para>
-<screen>$ sudo swift-recon --diskusage
+<screen>&prompt.sudo;swift-recon --diskusage
 ===============================================================================
```

Comment:
sudo -i ?

Reply:
swift-recon --help 
runs okay as user=ardana

other swift-recon commands error with 
No such file or directory: '/etc/swift/object.ring.gz'

This needs to be checked on an install that has the necessary files

Reply:
on qa env 10.84.81.17

without sudo get an error like this
ardana@ardana-qe007-cp1-c1-m1-mgmt:~> swift-recon --md5 account
===============================================================================
Error: [swift-hash]: both swift_hash_path_suffix and swift_hash_path_prefix are missing from /etc/swift/swift.conf

with sudo it seems to work
ardana@ardana-qe007-cp1-c1-m1-mgmt:~> sudo swift-recon --md5 account
===============================================================================
--> Starting reconnaissance on 3 hosts (account)
===============================================================================
[2018-07-25 15:01:35] Checking ring md5sums
3/3 hosts matched, 0 error[s] while checking hosts.
===============================================================================
[2018-07-25 15:01:35] Checking swift.conf md5sum
3/3 hosts matched, 0 error[s] while checking hosts.
===============================================================================